### PR TITLE
Edited swagger definition of the devPortal 

### PR DESCRIPTION
**Pull request by thisaltennakoon**

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/ApisApiServiceImpl.java [hunk 1]
```diff
@@ -248,7 +248,14 @@ public Response addCommentToAPI(String apiId, CommentDTO body, MessageContext me
...
             }
-            Comment comment = CommentMappingUtil.fromDTOToComment(body, username, apiId);
+            Comment comment = new Comment();
+            comment.setText(addCommentDTO.getContent());
+            comment.setCategory(addCommentDTO.getCategory());
+            comment.setParentCommentID(parentCommentID);
+            comment.setEntryPoint("devPortal");
+            comment.setUser(username);
+            comment.setApiId(apiId);
+            //Comment comment = CommentMappingUtil.fromAddCommentDTOToComment(addCommentDTO, parentCommentID, username, apiId);
```
Comment (wasuradananjith) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/ApisApiServiceImpl.java:258]: Is there a reason to keep this commented code? Or else, shall we remove it?

File: components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIManager.java [hunk 2]
```diff
@@ -19,23 +19,7 @@
...
-import org.wso2.carbon.apimgt.api.model.DocumentationContent;
-import org.wso2.carbon.apimgt.api.model.DocumentationType;
-import org.wso2.carbon.apimgt.api.model.Identifier;
-import org.wso2.carbon.apimgt.api.model.Mediation;
-import org.wso2.carbon.apimgt.api.model.ResourceFile;
-import org.wso2.carbon.apimgt.api.model.SubscribedAPI;
-import org.wso2.carbon.apimgt.api.model.Subscriber;
-import org.wso2.carbon.apimgt.api.model.Tier;
-import org.wso2.carbon.apimgt.api.model.Wsdl;
+import org.wso2.carbon.apimgt.api.model.*;
```
Comment (msm1992) [inline components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIManager.java:22]: Shall we revert this * import change ?

Comment (thisaltennakoon) [reply components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIManager.java:22]: Already fixed

File: components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/Comment.java [hunk 3]
```diff
@@ -18,56 +18,65 @@
 
 package org.wso2.carbon.apimgt.api.model;
 
+import java.util.*;
```
[the remark this answers is not in the document]
Comment (thisaltennakoon) [reply components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/Comment.java:21]: Already fixed

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/dto/CommentDTO.java [hunk 4]
```diff
@@ -0,0 +1,297 @@
+package org.wso2.carbon.apimgt.rest.api.publisher.v1.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import org.wso2.carbon.apimgt.rest.api.publisher.v1.dto.CommentListDTO;
+import org.wso2.carbon.apimgt.rest.api.publisher.v1.dto.CommenterInfoDTO;
+import javax.validation.constraints.*;
```
[the remark this answers is not in the document]
Comment (thisaltennakoon) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/dto/CommentDTO.java:9]: This file is a generated file. Even though these '*' type imports can be removed, these will be added again and again when we build the component.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java [hunk 5]
```diff
@@ -58,45 +58,9 @@
...
-import org.wso2.carbon.apimgt.api.model.APIRevisionDeployment;
-import org.wso2.carbon.apimgt.api.model.APIStateChangeResponse;
-import org.wso2.carbon.apimgt.api.model.APIStatus;
-import org.wso2.carbon.apimgt.api.model.APIStore;
-import org.wso2.carbon.apimgt.api.model.ApiTypeWrapper;
-import org.wso2.carbon.apimgt.api.model.BlockConditionsDTO;
-import org.wso2.carbon.apimgt.api.model.DeploymentEnvironments;
-import org.wso2.carbon.apimgt.api.model.DeploymentStatus;
-import org.wso2.carbon.apimgt.api.model.Documentation;
+import org.wso2.carbon.apimgt.api.model.*;
```
[the remark this answers is not in the document]
Comment (thisaltennakoon) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java:61]: Fixed in https://github.com/wso2/carbon-apimgt/pull/11948

**Pull request merged**